cli/cmd/inject.go

```diff
@@ -304,6 +304,127 @@ func injectPodTemplateSpec(t *v1.PodTemplateSpec) enhancedPodTemplateSpec {
 	}
 }
 
+func printSideCar() string {
```

Comment:
I suggest we find a way to share the exact same code between the inject and install commands. i.e. have install generate the serialized YAML and then have inject parse it, exactly replicating the `conduit install | conduit inject` shell pipe sequence.

In particular, it isn't obvious here whether or how the proxy is configured differently in the control plane than it is in the data plane. That's something that should be obvious in this PR.

Reply:
Oops, sorry @briansmith I didn't clean up my code. There is an `injectYAML` func [here](https://github.com/runconduit/conduit/blob/54154c1bc7077b3df36d46a5750324384bc782b5/cli/cmd/inject.go#L383) that is setup for that exact scenario.

Reply:
> Oops, sorry @briansmith I didn't clean up my code. There is an injectYAML func here that is setup for that exact scenario.

OK, but you didn't modify the existing code for `conduit inject` to use `injectYAML` at https://github.com/runconduit/conduit/blob/54154c1bc7077b3df36d46a5750324384bc782b5/cli/cmd/inject.go#L78-L99 so it isn't clear what the difference between the two code paths is. I expect that the `injectYAML` function can and should be directly factored out of the existing inject.go code.

Reply:
Ahh thats my bad. I agree with you, I should refactor the inject.go. I will add that to a follow up commit.